stargazer/stargazer.py

```diff
@@ -130,6 +131,7 @@ def extract_model_data(self, model):
                            'cov_std_err' : 'bse',
                            'r2' : 'rsquared',
                            'r2_adj' : 'rsquared_adj',
+                           'p_r2_adj': 'prsquared',
```

Comment:
Thanks for your PR!

Are you sure what you're getting is "adjusted"? The [docs](https://www.statsmodels.org/stable/generated/statsmodels.discrete.discrete_model.LogitResults.prsquared.html) just describe this is as "McFadden’s pseudo-R-squared".

Reply:
You're right! Should I change names in the variables?

Reply:
Yes, please! And also the labels.

Reply:
Done! :) Sorry for the delay, let me know if you have more comments ^^